manifests/init.pp

```diff
@@ -226,6 +226,10 @@
     $apache_https_chain = $certs::katello_server_ca_cert
   }
 
+  $api_client_auth_cn_map = Hash($foreman_proxy::trusted_hosts.map |String $host| {
```

Comment:
I think you're not required to use a data type so this also works.
```suggestion
  $api_client_auth_cn_map = Hash($foreman_proxy::trusted_hosts.map |$host| {
```

However, it's fine to keep it in.